backend/voting/admin.py

```diff
@@ -51,7 +51,8 @@ class Media:
 
 
 @admin.register(RankSubmission)
-class RankSubmissionAdmin(admin.ModelAdmin):
+class RankSubmissionAdmin(AdminUsersMixin):
+    user_fk = "speaker_id"
```

Comment:
where is speaker_id coming from?

```
Cannot resolve keyword 'speaker_id' into field. Choices are: absolute_rank, absolute_score, id, rank_request, rank_request_id, submission, submission_id, topic_rank
```

we are getting this error now 🤔

Reply:
ok so RankSubmission has submission, so speaker_id doesn't exist but `submission__speaker_id` does

Reply:
I am not sure how it worked for you locally 🤔

Reply:
I did a little change as the `speaker_id` is in the `submission`... can you try again? tnks

Reply:
you have to change user_fk to `submission__speaker_id` otherwise it will just fail when trying to fetch the list of users